test/performance/dataplane-probe/dataplane-probe.yaml

```diff
@@ -68,12 +68,12 @@ spec:
             - name: GOOGLE_APPLICATION_CREDENTIALS
               value: /var/secret/robot.json
             volumeMounts:
-            - name: service-account
+            - name: mako-secrets
```

Comment:
Please change the instructions, which reference the old name

Reply:
Fixed. 
A few links are invalid for now, e.g. https://github.com/knative/serving/blob/master/test/performance/dataplane-probe/dataplane-probe.yaml#L47, since the line moved.
I will update them in a followup PR when I add alerts for other benchmarks.